docs/content/en/docs/release-notes/_index.md

```diff
@@ -0,0 +1,95 @@
+
+
+## K8ssandra 1.1.0 changes
+
+ *(DRAFT comment: in CHANGELOG-110.md, can we link to the issue numbers (as done below), and can each issue on GH then link to relevant commit IDs?)*
+
+* Remove Jolokia integration [#533](https://github.com/k8ssandra/k8ssandra/issues/533)
+* Upgrade to medusa-operator 0.2.0 [#630](https://github.com/k8ssandra/k8ssandra/issues/630)
+* Mount Cassandra pod labels in volume [#613](https://github.com/k8ssandra/k8ssandra/issues/613)
+* Shut down cluster by default with in-place restores [#611](https://github.com/k8ssandra/k8ssandra/issues/611)
+
+## K8ssandra 1.1.0 enhancements
+
+* Add option to disable Cassandra logging sidecar [#576](https://github.com/k8ssandra/k8ssandra/issues/576)
+* Upgrade Reaper to 2.2.2 and Medusa to 0.9.1 [#530](https://github.com/k8ssandra/k8ssandra/issues/530)
+* Split dashboards into separate configmaps [#504](https://github.com/k8ssandra/k8ssandra/issues/504)
+* Upgrade Stargate to 1.0.11, and add a `preStop` lifecycle hook to improve behavior when reducing the number of Stargate replicas in the presence of live traffic [#436](https://github.com/k8ssandra/k8ssandra/issues/436)
+* Add automation for stable and next release streams [#419](https://github.com/k8ssandra/k8ssandra/issues/419)
+* Add support for `additionalSeeds` in the CassandraDatacenter [#547](https://github.com/k8ssandra/k8ssandra/issues/547)
+* Documentation updates and additions:
+  * Developer documentation [#239](https://github.com/k8ssandra/k8ssandra/issues/239)
+  * Sample values.yaml explanations and examples [#510](https://github.com/k8ssandra/k8ssandra/issues/510)
+  * S3-compliant MinIO buckets for Medusa backup and restore operations, and related edits for the separate Amazon S3 topic [#556](https://github.com/k8ssandra/k8ssandra/issues/556)
+  * Migrating existing Cassandra to K8ssandra [#377](https://github.com/k8ssandra/k8ssandra/issues/377) 
+  * Underlying considerations and impacts of scaling nodes up or down in Kubernetes clusters [#501](https://github.com/k8ssandra/k8ssandra/issues/501) 
+
+## K8ssandra 1.1.0 bug fixes
+
+* Cassandra config clobbering when enabling Medusa [#475](https://github.com/k8ssandra/k8ssandra/issues/475)
+* cqlsh commands show warnings [#396](https://github.com/k8ssandra/k8ssandra/issues/396)
+* Fix issue with scripts not being checked out before attempting to run them [#516](https://github.com/k8ssandra/k8ssandra/issues/516)
+* Remove GitHub Actions for pre-releasing off of main [#517](https://github.com/k8ssandra/k8ssandra/issues/517)
+* Fix Cassandra config clobbering when enabling Medusa [#475](https://github.com/k8ssandra/k8ssandra/issues/475)
+* Create cass-operator webhook secret [#590](https://github.com/k8ssandra/k8ssandra/issues/590)
+* helm uninstall can leave CassandraDatacenter behind [#623](https://github.com/k8ssandra/k8ssandra/issues/623)
+* Fix indentation error in backup-restore-values.yaml [#602](https://github.com/k8ssandra/k8ssandra/issues/602)
+
```

Comment:
This bug is also going to be merged for 1.1 and we'll want to include it:

https://github.com/k8ssandra/k8ssandra/issues/616

Reply:
Added in next commit

Reply:
Done